libs/zgui/src/gui.zig

```diff
@@ -1450,6 +1450,50 @@ pub fn combo(label: [:0]const u8, args: Combo) bool {
         args.popup_max_height_in_items,
     );
 }
+/// creates a combo box directly from a pointer to an enum value using zig's
+/// comptime mechanics to infer the items for the list at compile time
+pub fn comboFromEnum(
+    label: [:0]const u8,
+    /// must be a pointer to an enum value (var my_enum: *FoodKinds = .Banana)
+    /// that is backed by some kind of integer that can safely cast into an
+    /// i32 (the underlying imgui restriction)
+    current_item: anytype
```

Comment:
should add a comma after this last argument otherwise zig fmt will do some weird stuff

Reply:
Cool, good to know.  We don't use `zig fmt` so this wasn't on my radar.